src/three-components/SmoothControls.ts

```diff
@@ -226,13 +247,11 @@ export class SmoothControls extends EventDispatcher {
     this[$onTouchMove] = (event: Event) =>
         this[$handlePointerMove](event as TouchEvent);
 
-    this[$previousPosition].copy(this.camera.position);
-    this[$positionToSpherical](this.camera.position, this[$spherical]);
-    this[$targetSpherical].copy(this[$spherical]);
-
     this[$options] = Object.assign({}, DEFAULT_OPTIONS);
 
-    this.setOrbit();
+    this.setOrbit(0, Math.PI / 2, 1);
```

Comment:
Previously, initial orbital position was inferred from the configured camera's position. One reason why this seems preferable is that it gives the user the ability to change camera control schemes without forcing the camera to reset its position to some fixed default.

OTOH this is of marginal benefit to `<model-viewer>`. But, the fixed defaults used here seem separable from `SmoothControls` implementation...

Reply:
I removed that because I thought it was confusing. Sure, it uses the camera's existing position, but then ignores its orientation and instead points at the target, which you were not allowed to configure yet. Plus the most common case is that the camera is at the origin, which leads to a divide by zero, which was only solved by some surprising side effects of the default minimum radius. I think this will be more clear for developers.